config/environment.js

```diff
@@ -95,6 +95,9 @@ module.exports = function(environment) {
   };
 
   if (environment === 'development') {
+    ENV['ember-cli-mirage'] = {
+      enabled: false,
```

Comment:
do you want to commit this change ?

Reply:
@julialucyhogan yes, because people tend to use `ember s` (develpment environment` oto check up on map styles, visual stuff.